public/app/features/dashboard-scene/settings/VersionsEditView.tsx

```diff
@@ -1,36 +1,125 @@
 import React from 'react';
 
-import { PageLayoutType } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase } from '@grafana/scenes';
+import { PageLayoutType, dateTimeFormat, dateTimeFormatTimeAgo } from '@grafana/data';
+import { SceneComponentProps, SceneObjectBase, sceneGraph } from '@grafana/scenes';
 import { Page } from 'app/core/components/Page/Page';
+import {
+  DecoratedRevisionModel,
+  VersionsHistorySpinner,
+} from 'app/features/dashboard/components/DashboardSettings/VersionsSettings';
+import { RevisionsModel, VersionHistoryTable, historySrv } from 'app/features/dashboard/components/VersionHistory';
```

Comment:
I suggest moving those components and types under `dashboard-scene` directory scope. When we decide to remove the legacy code, we will already have things where they should belong.

Reply:
Went on and moved/refactored where possible all of the Versions History components since they will all be used in later stages of this work (e.g: comparison).